framework/docker/da_node.go

```diff
@@ -178,20 +176,20 @@ func (n *DANode) startNode(ctx context.Context, opts types.DANodeStartOptions, e
 }
 
 // initNode initializes the DANode by running the "init" command for the specified DANode type, network, and keyring settings.
-func (n *DANode) initNode(ctx context.Context, env []string) error {
-	// note: my_celes_key is the default key name for the bridge node.
-	cmd := []string{"celestia", n.nodeType.String(), "init", "--p2p.network", n.cfg.ChainID, "--keyring.keyname", "my_celes_key", "--node.store", n.homeDir}
+func (n *DANode) initNode(ctx context.Context, chainID string, env []string) error {
+	// note: my_celes_key is the default key name for the da node.
+	cmd := []string{"celestia", n.nodeType.String(), "init", "--p2p.network", chainID, "--keyring.keyname", "my_celes_key", "--node.store", n.homeDir}
```

Comment:
the same can be applied to `init` args if required. For now we are just making start args fully flexible